src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Collection/DictionaryDefaultConverter.cs

```diff
@@ -297,7 +297,7 @@ TKey ReadDictionaryKey(ref Utf8JsonReader reader, ref ReadStack state)
             return success;
         }
 
-        public sealed override void CreateInstance(ref Utf8JsonReader reader, ref ReadStack state, JsonSerializerOptions options)
+        internal sealed override void CreateInstanceForReferenceResolver(ref Utf8JsonReader reader, ref ReadStack state, JsonSerializerOptions options)
```

Comment:
The type is already internal, so this should be left as public - here and elsewhere.

Reply:
This method is inherited from `JsonConverter` which is public, if I make it public here and in the base, then we would be adding new API to the base type.